test-fixtures/candidate-search/query-list/bib/standardIdentifiers/06/metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-  "description": "DISABLED: Should not generate 'undefined' standard identifier query (ISSN with colon)",
-  "enabled": false,
+  "description": "Should not generate 'undefined' standard identifier query (ISSN with colon)",
+  "enabled": true,
```

Comment:
No need to add this if there is enabled = true default value in fixugen callback

Reply:
Removed.